tools/building.py

```diff
@@ -1579,7 +1579,7 @@ def run_binaryen_command(tool, infile, outfile=None, args=[], debug=False, stdou
       if shared.Settings.SUPPORT_LONGJMP:
         extra += '\nnote: to disable longjmp support (which requires changes after link) use -s SUPPORT_LONGJMP=0'
       if shared.Settings.OPT_LEVEL > 0:
-        extra += '\nnote: optimizations always require changes, build with -O0 instead'
+        extra += '\nnote: -O2+ optimizations always require changes, build with -O0 or -O1 instead'
```

Comment:
drive-by for lime 1578: it would be more clear to a non-emscripten developer if it said 'int64' legalization or something that mentioned that specifically.

Reply:
Good idea, added.